proto/viam/app/data/v1/data.proto

```diff
@@ -203,9 +203,12 @@ message TabularDataBySQLResponse {
 // TabularDataByMQLRequest requests tabular data using an MQL query.
 message TabularDataByMQLRequest {
   string organization_id = 1;
-  // mql_query accepts a MongoDB aggregation pipeline, to be run on the "sensorData.readings"
-  // namespace, which holds the Viam organization's tabular data.
+  // mql_query is deprecated.
   string mql_query = 2;
+  // mql_binary accepts a MongoDB aggregation pipeline as a list of BSON documents, where each
```

Comment:
[nit] list of serialized BSON documents?

Reply:
I think simply saying "BSON documents" is actually more accurate, since BSON is already in a binary/byte format - a BSON document is just a series of bytes with a particular format. You can serialize/encode objects into BSON, but saying "serialized BSON documents" to me makes it sound like we took BSON documents and serialized/encoded them further. Lmk if you disagree!

Reply:
Good point! Agreed.